rust/platform/triple_mappings.bzl

```diff
@@ -42,6 +78,7 @@ _SYSTEM_TO_BUILTIN_SYS_SUFFIX = {
 }
 
 _SYSTEM_TO_BINARY_EXT = {
+    "android": "",
```

Comment:
This is not related to this PR, could you please split it into a standalone PR with a focused description? Not sure there's a good way to add a test for it, but maybe you'll come up with something?

Reply:
This is actually a gap in support from whenever `android` was added to the list of supported systems:
https://github.com/bazelbuild/rules_rust/blob/72c591d51311e3eb829220a49cc00700e8a6dc2b/rust/platform/platform.bzl#L54-L61

I definitely don't have bandwidth to setup an android test so I'd say either this would go in as is or support for android be dropped until a test can be added. Thoughts?

Reply:
I'm fine with merging without a test, but please still create a separate PR for that.

Do we support android now? I honestly don't know. And I don't know if we have a single android test in the repo. And I'd say if we don't have a test, then we don't really support it :)

Reply:
We have no test 😞 

Creating a separate PR though

Reply:
https://github.com/bazelbuild/rules_rust/pull/813